packages/cli/commands/project/install.js

```diff
@@ -0,0 +1,175 @@
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const { getProjectConfig } = require('../../lib/projects');
+const path = require('node:path');
+const { walk } = require('@hubspot/local-dev-lib/fs');
+const { execSync } = require('child_process');
+const { promptUser } = require('../../lib/prompts/promptUtils');
+const chalk = require('chalk');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
+
+function isGloballyInstalled(command) {
+  try {
+    execSync(`${command} --version`);
+    return true;
+  } catch (e) {
+    return false;
+  }
+}
+
+const YARN = 'yarn';
+const NPM = 'npm';
+const PNPM = 'pnpm';
+
+const packageManagers = {
+  [NPM]: 'package-lock.json',
+  [PNPM]: 'pnpm-lock.yaml',
+  [YARN]: 'yarn.lock',
+};
```

Comment:
I am still holding out hope that [corepack](https://nodejs.org/api/corepack.html) turns into the dream tool I thought it was going to be. Not saying we slow things down for that or anything, just we should keep an eye on that since that's part of node core and just feels like it HAS TO HAPPEN at some point, and become something people actually use.

That would eventually give us the benefit of enforcing the same package manager at both build time and local dev too, along with error messages if anyone tries to run a command from another package manager than the one selected for the project.

I'm praying this works for us someday!!

Reply:
`corepack` looks cool and would really help to enforce consistency.  It's been lingering in `experimental` for a while.  I really hope it makes it to stable someday 🤞 

@brandenrodgers and I talked a little bit about making this configurable at the project or app level even.  We still need to think about what that would look like.  We could possibly use that information to mimic the validations that `corepack` is doing, but on `hs project upload`.  I.E. they have `yarn` set as their package manager for the project, but we detect a `pnpm` lockfile and log a warning.